Apps/WebClient/src/ClientApp/src/router.ts

```diff
@@ -62,15 +62,19 @@ enum UserState {
     registered = "registered",
     pendingDeletion = "pendingDeletion",
     invalidLogin = "invalidLogin",
+    offline = "offline",
 }
 
 function calculateUserState() {
+    const isOffline = store.getters["config/isOffline"];
     const isAuthenticated: boolean = store.getters["auth/oidcIsAuthenticated"];
     const isValid: boolean = store.getters["auth/isValidIdentityProvider"];
     const isRegistered: boolean = store.getters["user/userIsRegistered"];
     const userIsActive: boolean = store.getters["user/userIsActive"];
 
-    if (!isAuthenticated) {
+    if (isOffline) {
```

Comment:
Is the whitelist supported?

Reply:
it is supported. The Vuex method checks it